make/config/samplewebhook/sample/main.go

```diff
@@ -123,7 +123,7 @@ func (c *customDNSProviderSolver) CleanUp(ch *v1alpha1.ChallengeRequest) error {
 }
 
 // Initialize will be called when the webhook first starts.
-// This method can be used to instantiate the webhook, i.e. initialising
+// This method can be used to instantiate the webhook, i.e. initializing
```

Comment:
This repository is actually fairly heavily en-US for a bunch of things (there were a couple of things w/ 0 instances that I left w/ `-ising`, but...)